Jenkinsfile

```diff
@@ -759,6 +759,69 @@ pipeline {
             }
         }
 
+
+        // Functional tests using BDD Stack
+        // See https://github.com/BCDevOps/BDDStack
+        stage('TEST - Smoke Tests') {
+            when {
+                expression { env.CHANGE_TARGET == 'master' }
```

Comment:
I think we should run this in test and in production. I realize the smoke tests run as part of the normal tests, so that would mean any tests we run here, would run twice - but any issues in this section of Jenkins would then be exposed during test.

Reply:
Not a bad idea!  Will make the addition.